packages/firefuel/test/src/firefuel_collection_test.dart

```diff
@@ -8,13 +8,13 @@ import '../utils/test_user.dart';
 const _testUsersCollectionName = 'testUsers';
 
 void main() {
-  late FirebaseFirestore instance;
   late TestCollection testCollection;
   final defaultUser = TestUser('testName');
 
   setUp(() {
-    instance = FakeFirebaseFirestore();
-    testCollection = TestCollection(instance);
+    final instance = FakeFirebaseFirestore();
+    Firefuel.inititialize(instance);
```

Comment:
This is only used once, no need for the variable
```suggestion
    Firefuel.inititialize(FakeFirebaseFirestore());
```